pkgs/collection/lib/src/iterable_zip.dart

```diff
@@ -13,7 +13,9 @@ import 'dart:collection';
 /// combined into a single list, which becomes the next value of this
 /// [Iterable]'s [Iterator]. As soon as any of the iterators run out,
 /// the zipped iterator also stops.
-class IterableZip<T> extends IterableBase<List<T>> {
+/// @nodoc
+@Deprecated('Use [i1, i2].zip from dart:collection')
+final class IterableZip<T> extends IterableBase<List<T>> {
```

Comment:
should we just do another "dot" release to deprecate and then drop this? Let's double down on the breaking changes.